tests/e2e/specs/layerDetails.spec.js

```diff
@@ -0,0 +1,35 @@
+/// <reference types="cypress" />
+
+describe('Test layer menu show details button', () => {
+    it('Toggling layer details menu', () => {
+        cy.goToMapView('en', {
+            layers: 'test.wms.layer;test.wmts.layer',
+        })
+        // clicking on the menu button
+        cy.get('[data-cy="menu-button"]').click()
+        // get and click the wms layer detail button
+        cy.get('[data-cy="button-open-visible-layer-settings-test.wms.layer"]')
+            .as('wmsDetailsBtn')
+            .click()
+        cy.get('@wmsDetailsBtn')
+            .parents('[class="menu-layer-list-item"]')
+            .find('[class="menu-layer-list-item-details"]')
+            .as('wmsLayerDetail')
+            .should('be.visible')
+        // the other layer details should not be visible
+        cy.get('[data-cy="button-open-visible-layer-settings-test.wmts.layer"]')
+            .as('wmtsDetailBtn')
+            .parents('[class="menu-layer-list-item"]')
+            .find('[class="menu-layer-list-item-details"]')
+            .as('wmtsLayerDetail')
+            .should('not.be.visible')
```

Comment:
I'd prefer that we don't use CSS class selector, but instead that we add a custom HTML attribute `data-cy="..."` to any element we need to grab in our test (see [Cypress's best practices on that](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements))

Reply:
Good idea, thanks for the tip I'll change that.